server/functions/framework/interpreter_operation.go

```diff
@@ -0,0 +1,77 @@
+	OpCode_Exception                  // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-ERROR-TRAPPING
+	OpCode_Exit                       // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONTROL-STRUCTURES-LOOPS
+	OpCode_Execute                    // https://www.postgresql.org/docs/15/plpgsql-statements.html#PLPGSQL-STATEMENTS-EXECUTING-DYN
+	OpCode_For                        // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONTROL-STRUCTURES-LOOPS
+	OpCode_Foreach                    // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONTROL-STRUCTURES-LOOPS
+	OpCode_Get                        // https://www.postgresql.org/docs/15/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
+	OpCode_If                         // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONDITIONALS
+	OpCode_InsertInto                 // https://www.postgresql.org/docs/15/plpgsql-statements.html
+	OpCode_Loop                       // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONTROL-STRUCTURES-LOOPS
+	OpCode_Perform                    // https://www.postgresql.org/docs/15/plpgsql-statements.html
+	OpCode_Query                      // This is just a standard query, nothing special
+	OpCode_Return                     // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-STATEMENTS-RETURNING
+	OpCode_ScopeBegin                 // This is used for scope control, specific to Doltgres
+	OpCode_ScopeEnd                   // This is used for scope control, specific to Doltgres
+	OpCode_ScopeRestart               // This is used for scope control, specific to Doltgres
+	OpCode_SelectInto                 // https://www.postgresql.org/docs/15/plpgsql-statements.html
+	OpCode_When                       // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONDITIONALS
+	OpCode_While                      // https://www.postgresql.org/docs/15/plpgsql-control-structures.html#PLPGSQL-CONTROL-STRUCTURES-LOOPS
+	OpCode_UpdateInto                 // https://www.postgresql.org/docs/15/plpgsql-statements.html
+)
+
+// IterationType represents how iteration should proceed. Each type has a different ruleset that it follows, and all
+// forms of control flow can be represented by one of these types.
+type IterationType uint8
+
+const (
+	// IterationType_Normal :: Every statement encountered should be executed. If a conditional continuation is
+	// encountered (ELSIF or ELSE), then this turns into IterationType_SkipToEnd.
+	IterationType_Normal IterationType = iota
+	// IterationType_SkipToConditional :: Every statement should be ignored until a conditional is encountered. Only
+	// conditionals with the iterationID will be considered. This reverts to IterationType_Normal if a ScopeEnd with the
+	// iterationID is encountered.
+	IterationType_SkipToConditional
```

Comment:
This is more like `SkipToBlockEnd` right? Breaks when the current scope ends or you hit an `Else`

Reply:
A way better name would be `SkipToElse`, since this only applies to the `Else` opcode. It does break when the current scope ends, but I'll leave it such you have to match the ID, since it gives a bit more versatility without any additional complication.